### DISCUSSION TITLE
Can I import multiple cgns mesh files and export as a single su2 mesh file? (multizone mesh)

### DISCUSSION THREAD

**Discussion by koraysevinc** [nschloe/meshio#1000] [Q&A]
Hi

I have a  question. I have separate mesh files for multirow turbomachinery problem. Each rotor stator has their own separate mesh files in CGNS format.

SU2 doesn't read multiple mesh files. It needs to have a single mesh file but with multiple zones. Can I use meshio for this purpose?

Thanks.
Koray

Previously, I imported meshes into ICEM CFD and exported as a single file. But SU2 saw it as a single zone mesh. That approach didn't work for me.

**Comment by nschloe** [comment]
You could use Python to read all your CGNS files, and then combine the meshes into one to write them out to SU2. Not sure about _zones_ right now.
